### DISCUSSION TITLE
security concern

### DISCUSSION THREAD

**Discussion by tallpeece** [ezra-bible-app/ezra-bible-app#977] [Q&A] [answered]
Hello when installing the package it fails verification check due to package no being signed
how much of a security risk is this?

![image](https://github.com/ezra-bible-app/ezra-bible-app/assets/147283252/27d5d92f-e3dd-4814-b541-934655cbd03c)
![image](https://github.com/ezra-bible-app/ezra-bible-app/assets/147283252/f282d308-6765-45f2-b036-af4b61cf0db2)

**Comment by tobias-klein** [comment, marked as answer]
The Linux release packages are automatically generated based on GitHub Actions. Binary signing could enhance the confidence that the package is coming from a known source. The same warning is shown on Windows.

For the Windows package I am actually thinking about getting the necessary certificate. However, it costs money, roughly ~150$ per year.

I haven't researched yet what effort it would be to implement the signing process for Linux.
